str.go

```diff
@@ -13,14 +19,17 @@ import (
 // STR should be final
 type SignedTreeRoot struct {
 	tree        *MerkleTree
-	counter     int64
-	prevCounter int64
+	counter     uint64
```

Comment:
I would still call these ``epoch`` and ``prevEpoch`` for clarity and consistency with the spec.